src/foam/u2/view/StrategizerRichChoiceView.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * @license
+ * Copyright 2025 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.view',
+  name: 'StrategizerRichChoiceView',
+  extends: 'foam.u2.view.RichChoiceView',
+
+  documentations: 'A rich choice view that gets its choices array from Strategizer.',
+
+  imports: [
+    'strategizer'
+  ],
+
+  properties: [
+    {
+      class: 'String',
+      name: 'desiredModelId',
+      required: true
+    },
+    {
+      class: 'String',
+      name: 'target'
+    },
+    {
+      name: 'search',
+      value: true
+    },
+    {
+      name: 'sections',
+      factory: function() {
+        return [
+          {
+            dao: foam.dao.ArrayDAO.create({
+              of: "foam.mlang.LabeledValue"
+            }),
+            searchBy: [
+              foam.mlang.LabeledValue.LABEL
+            ]
+          }
+        ];
+      }
+    },
+    {
+      name: 'choosePlaceholder',
+      expression: function(desiredModelId) {
+        let cls = foam.maybeLookup(desiredModelId);
+        return this.CHOOSE_FROM + ' ' + cls.model_.plural + '...';
+      }
+    }
+  ],
+
+  methods: [
+    async function init() {
+      this.SUPER();
+      var self = this;
+      const strategyReferences = await self.strategizer.query(null, self.desiredModelId, self.target);
+      const choices = strategyReferences
+        .reduce((arr, sr) => {
+          if ( ! sr.strategy ) {
+            console.warn('Invalid strategy reference: ' + sr.id);
+            return arr;
+          }
+          let value = foam.mlang.LabeledValue.create(
```

Comment:
require foam.mlang.LabeledValue, then
```suggestion
          let value = this.LabeledValue.create(
```